packs/data/extinction-curse-bestiary.db/lyrt-cozurn.json

```diff
@@ -1289,20 +1391,24 @@
                     "value": 1
                 },
                 "description": {
-                    "value": "<p>Lyrt possesses a Large or smaller object within 20 feet, making it an animated object. This animated object's level can be no higher than Lyrt's level - 2. If the target object is being held by a creature, the bearer can attempt a Will save to prevent the possession. This possession ends when the object is destroyed or Lyrt leaves it. At this point, Lyrt reappears in the object's square and can't Inhabit an Object again for [[/br 1d4 #rounds]]{1d4 rounds}.</p>"
+                    "value": "<p>Lyrt possesses an object of size Large or smaller within 20 feet, making it an animated object. This animated object's level can be no higher than Lyrt's level -2. If the target object is being held by a creature, the bearer can attempt a @Check[type:will|dc:34] save to prevent the possession.</p>\n<p>This possession ends when the object is destroyed or Lyrt leaves it. At this point, Lyrt reappears in the object's square and can't Inhabit an Object again for [[/br 1d4 #rounds]]{1d4 rounds}.</p>\n<p><em><strong>Dev Note:</strong> The book does not provide a DC for this ability. We have used the DC by Level table to provide a functional DC.</em></p>"
```

Comment:
```suggestion
                    "value": "<p>Lyrt possesses an object of size Large or smaller within 20 feet, making it an animated object. This animated object's level can be no higher than Lyrt's level -2. If the target object is being held by a creature, the bearer can attempt a @Check[type:will|dc:34] save to prevent the possession.</p>\n<p>This possession ends when the object is destroyed or Lyrt leaves it. At this point, Lyrt reappears in the object's square and can't Inhabit an Object again for [[/br 1d4 #Inhabit an Object Recharge]]{1d4 rounds}.</p>\n<p><em>Note: A DC was not provided for this ability by Paizo. The DC present here is a moderate DC for the creature level according to the Gamemastery Guide creature building Tables.</em></p>"
```